Formula/giter8.rb

```diff
@@ -7,7 +7,10 @@ class Giter8 < Formula
 
   bottle :unneeded
 
-  depends_on java: "1.8+"
+  # Doesn't actually build from source
```

Comment:
What do you mean by "build from source"?

It's `bottle :unneeded` formula, and we don't build such kind of formulae usually

Reply:
Yeah, that's bad naming. It downloads a shell script that downloads the actual software and dependencies. The change in https://github.com/Homebrew/homebrew-core/pull/50160 shows what the script does